src/compositions/FieldsetHeader/FieldsetHeader.stories.tsx

```diff
@@ -38,9 +38,9 @@ export const Template: Story<FieldsetHeaderProps> = (args: FieldsetHeaderProps)
     return <FieldsetHeader {...args} active={active} onClick={() => setActive(!active)} />;
 };
 
-export const withIcon = Template.bind({});
+export const withDecorator = Heading.bind({});
 
-withIcon.args = {
+withDecorator.args = {
```

Comment:
You could combine this by adding a boolean control to display/hide the decorator. I've done the same in the MenuItem story so you can just toggle the icon